container-storage-setup.sh

```diff
@@ -21,40 +21,42 @@
 
 set -e
 
-# container-storage-setup version information
-CSS_MAJOR_VERSION="0"
-CSS_MINOR_VERSION="1"
-CSS_SUBLEVEL="0"
-CSS_EXTRA_VERSION=""
+set_internal_gloabls() {
+  # container-storage-setup version information
+  CSS_MAJOR_VERSION="0"
+  CSS_MINOR_VERSION="1"
+  CSS_SUBLEVEL="0"
+  CSS_EXTRA_VERSION=""
 
-CSS_VERSION="${CSS_MAJOR_VERSION}.${CSS_MINOR_VERSION}.${CSS_SUBLEVEL}"
-[ -n "$CSS_EXTRA_VERSION" ] && CSS_VERSION="${CSS_VERSION}-${CSS_EXTRA_VERSION}"
+  CSS_VERSION="${CSS_MAJOR_VERSION}.${CSS_MINOR_VERSION}.${CSS_SUBLEVEL}"
+  [ -n "$CSS_EXTRA_VERSION" ] && CSS_VERSION="${CSS_VERSION}-${CSS_EXTRA_VERSION}"
 
-# This section reads the config file $INPUTFILE
-# Read man page for a description of currently supported options:
-# 'man container-storage-setup'
+  # This section reads the config file $INPUTFILE
+  # Read man page for a description of currently supported options:
+  # 'man container-storage-setup'
 
-DOCKER_ROOT_LV_NAME="docker-root-lv"
-DOCKER_ROOT_DIR="/var/lib/docker"
-DOCKER_METADATA_DIR="/var/lib/docker"
-DOCKER_ROOT_VOLUME_SIZE=40%FREE
+  DOCKER_ROOT_LV_NAME="docker-root-lv"
+  DOCKER_ROOT_DIR="/var/lib/docker"
+  DOCKER_METADATA_DIR="/var/lib/docker"
+  DOCKER_ROOT_VOLUME_SIZE=40%FREE
 
-STORAGE_IN_FILE="/etc/sysconfig/docker-storage-setup"
-STORAGE_OUT_FILE="/etc/sysconfig/docker-storage"
-STORAGE_DRIVERS="devicemapper overlay overlay2"
+  STORAGE_IN_FILE="/etc/sysconfig/docker-storage-setup"
```

Comment:
I looked at your code again, and it is ok.  I thought you were setting this after you set
STORAGE_IN_FILE=$1
STORAGE_OUT_FILE=$2